tools/matchms/matchms_convert.xml

```diff
@@ -0,0 +1,64 @@
+<tool id="matchms_convert" name="matchms convert" version="@TOOL_VERSION@+galaxy0" python_template_version="3.7">
+    <description>convert mass spectral library</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="creator"/>
+    <requirements>
+        <requirement type="package" version="@TOOL_VERSION@">matchms</requirement>
+    </requirements>
+
+    <command detect_errors="aggressive"><![CDATA[
+        python ${matchms_python_cli}
```

Comment:
single quotes

Reply:
resolved in https://github.com/RECETOX/galaxytools/pull/339/commits/0ab7ae8440f67a2d2ebc3c2d68740dbb934e286e